en/installation/debian.md

```diff
@@ -165,4 +165,12 @@ chromedriver --version
 
 You should receive an output with the latest version of ChromeDriver. If that's the case, you're good to go!
 
+If you are using an Arch-based distro, installing `chromium` from the `extra` repository should be sufficient.
```

Comment:
Why do we mention Arch in the instructions to install CONSUL on Debian? :thinking: